crypto/crypto_entropy.c

```diff
@@ -58,6 +60,15 @@ instantiate(void)
 	/* Mix the random seed into the state. */
 	update(seed_material, 48);
 
+#ifdef CPUSUPPORT_X86_RDRAND
+	/* Add output of RDRAND into the state. */
+	if (cpusupport_x86_rdrand()) {
+		if (generate_seed_rdrand(seed_material, 48))
+			return (-1);
+		update(seed_material, 48);
```

Comment:
There's no need for this to be 48 bytes.  The initial seeding is 48 bytes, but that's 48 bytes of "guaranteed" entropy, whereas RDRAND only gives us "probably" entropy anyway.

If we make this 32 bytes you can take this and the code you added to reseed() and make them a single `update_from_rdrand` function which has an internal `int buf[8]`.

(This also avoids the alignment issues -- `seed_material` is not guaranteed to be aligned properly for converting into `int *`.)